akka-persistence-typed/src/main/scala/akka/persistence/typed/scaladsl/Effect.scala

```diff
@@ -0,0 +1,78 @@
+/**
+ * Copyright (C) 2017-2018 Lightbend Inc. <https://www.lightbend.com>
+ */
+package akka.persistence.typed.scaladsl
+
+import akka.actor.typed.scaladsl.Behaviors
+import akka.annotation.DoNotInherit
+import akka.persistence.typed.internal._
+
+import scala.collection.{ immutable ⇒ im }
+
+/**
+ * Factories for effects - how a persistent actor reacts on a command
+ */
+object Effect {
+
+  // TODO docs
+  def persist[Event, State](event: Event): Effect[Event, State] = Persist(event)
+
+  // TODO docs
+  def persist[Event, A <: Event, B <: Event, State](evt1: A, evt2: B, events: Event*): Effect[Event, State] =
+    persist(evt1 :: evt2 :: events.toList)
+
+  // TODO docs
+  def persist[Event, State](eventOpt: Option[Event]): Effect[Event, State] =
+    eventOpt match {
+      case Some(evt) ⇒ persist[Event, State](evt)
+      case _         ⇒ none[Event, State]
+    }
+
+  // TODO docs
+  def persist[Event, State](events: im.Seq[Event]): Effect[Event, State] =
+    PersistAll(events)
+
+  // TODO docs
```

Comment:
some missing docs here

Reply:
Heh yeah, I added those todos, was missing since forever; Hoped to get this in and fix such things separately..

Reply:
Ok, that's fine by me